common/core/desktop/src/kmx/kmx_debugger.h

```diff
@@ -36,20 +37,22 @@ class KMX_DebugItems
 public:
   KMX_DebugItems(debug_items *items);
   void push_begin(km_kbp_state_debug_key_info *key_info, uint32_t flags);
-  void push_end(uint32_t flags);
-  void push_group_enter(LPGROUP group);
-  void push_group_exit(LPGROUP group, uint32_t flags);
-  void push_nomatch_enter(LPGROUP group);
-  void push_nomatch_exit(LPGROUP group);
-  void push_match_enter(LPGROUP group);
-  void push_match_exit(LPGROUP group);
+  void push_end(uint16_t first_action, uint32_t flags);
+  void push_group_enter(uint16_t first_action, LPGROUP group);
+  void push_group_exit(uint16_t first_action, uint32_t flags, LPGROUP group);
```

Comment:
Why `group` last here? I guess you could say it's always last, but it's always the second parameter except here.

Reply:
I was trying to match the order of struct members.